doc/nrf/known_issues.rst

```diff
@@ -552,6 +552,16 @@ Front-End Modules API implementation missing
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0
+
+DRGN-11963: LL control procedures cannot be initiated at the same time
+The following list of LL control procedures cannot be initiated at the same time:
+ * LE start encryption
+ * LE read remote features
+ * LE read remote version
+ * LE connection parameter update
+The controller will return HCI error code Controller Busy (0x3a).
+
```

Comment:
```suggestion

  **Workaround:** Do not initiate these procedures at the same time. 

```

Reply:
Added, not sure the application can prevent it when using the host (which is bad). But as a workaround for the controller it makes sense to have it.